crazyflie-link/src/context.rs

```diff
@@ -126,29 +118,38 @@ impl LinkContext {
     /// Send a packet to each URI and detect if an acknowledgement is sent back.
     ///
     /// Returns the list of URIs that acknowledged
-    pub async fn scan_selected(&self, uris: Vec<&str>) -> Result<Vec<String>> {
-        let mut found = Vec::new();
-        for uri in uris {
-            let (radio_nth, channel, address, _) = self.parse_uri(uri)?;
-            let radio = self.get_radio(radio_nth).await?;
-            let (ack, _) = radio
-                .send_packet_async(channel, address, vec![0xFF, 0xFF, 0xFF])
-                .await?;
-            if ack.received {
-                found.push(format!("{}{}", uri, "?safelink=0&ackfilter=0"));
-            }
-        }
-        Ok(found)
+    pub async fn scan_selected(&self, _uris: Vec<&str>) -> Result<Vec<String>> {
```

Comment:
This should likely be re-enabled in some way since without it the radio bootloader will not work

Reply:
Good find, see below.